priv/repo/migrations/20220829150240_create_tables.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Repo.Migrations.CreateTables do
```

Comment:
`:priv - the directory where to keep repository data, like migrations, schema and more. Defaults to "priv/YOUR_REPO". It must always point to a subdirectory inside the priv directory`
https://hexdocs.pm/ecto/Ecto.Repo.html

Reply:
I'm not 100% sure but I think It makes sense if the repo is a part of your application, while this is quite a specific, rare scenario where it's just testing util. `priv` is important for generating OTP release as it will contain the contents of `priv`, same goes for publishing a hex package. Here we don't want that to happen and these migrations should not be used outside of testing.